superset/utils/core.py

```diff
@@ -83,6 +83,7 @@
 from sqlalchemy.engine.reflection import Inspector
 from sqlalchemy.sql.type_api import Variant
 from sqlalchemy.types import TEXT, TypeDecorator
+from typing_extensions import TypedDict
```

Comment:
Importing from `typing_extensions` for backward compatibility as `TypeDict` is introduced in Python 3.8